integrations/langchain/pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
 name = "databricks-langchain"
-version = "0.3.0.dev0"
+version = "0.4.0.dev0"
```

Would be nice to make this one minor version above the last release version 
```suggestion
version = "0.4.1.dev0"
```